controllers/webhook/webhook.go

```diff
@@ -121,10 +121,12 @@ func (w *OpenDataHubWebhook) Handle(ctx context.Context, req admission.Request)
 		resp = w.checkDupCreation(ctx, req)
 	case admissionv1.Delete:
 		resp = w.checkDeletion(ctx, req)
+	case admissionv1.Update:
+		return admission.Allowed("")
 	default:
-		msg := fmt.Sprintf("No logic check by webhook is applied on %v request", req.Operation)
+		msg := fmt.Sprintf("No logic check by webhook is applied on %v request: allowed", req.Operation)
 		log.Info(msg)
-		resp = admission.Allowed("")
+		return admission.Allowed("")
```

Comment:
My logic behind Allowed("") was that it is a dummy value not returned but overwritten at the end (line 136 for the patch). Meant, the actual check can return own Denied message, there can be several checks and the first Denied will be reported, but if all are Allowed, the summary only is reported.

Do you prefer to change it to report blank message?

PS: does it make sense to remove UPDATE from the webhook configuration until we really use it?

Reply:
yeah, i think i overdid in this PR, the real part i would like is just to remove line `log.Info(msg)` from original code base.

P.S1:
not sure i understand "there can be several checks and the first Denied will be reported, but if all are Allowed, the summary only is reported"
you mean e.g  in the same "admissionv1.Create" will later add more checks, maybe even cascading ones?



P.S2: you mean to remove it from `verbs=create;update;delete`?
but we got log from pod: `No logic check by webhook is applied on UPDATE request` which means, we do have `req.Operation` on `admissionv1.Update`, right?

Reply:
two ways i can think of:
1. Keep 
```
operations:
    - CREATE
    - UPDATE
    - DELETE  
```
have switch-case on  Create, Delete, and default. 
but remove `msg := fmt.Sprintf("No logic check by webhook is applied on %v request", req.Operation) 
log.Info(msg)` from `default` block  => this is to clean up the log in pod.

2. remove UPDATE from operations
```
operations:
    - CREATE
    - DELETE  
```
remove "`default`" from switch-case. 
we only act on filters which is on create and delete, and the rest are all Allowed() by default. 

WDYT?

Reply:
> two ways i can think of:

Yes.

I personally prefer 2 a bit more, but both works for me.